docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -326,8 +326,10 @@ Exchange manager
 
 Exchange spooling is responsible for storing and managing spooled data for
 fault-tolerant execution. You can configure a filesystem-based exchange manager
-that stores spooled data in a specified location, such as an S3-compatible
-storage system, Google Cloud Storage (GCS), or a local filesystem.
+that stores spooled data in a specified location, such as :ref:`AWS S3
+<fte-exchange-aws-s3>` and S3-compatible systems, :ref:`Azure Blob Storage
+<fte-exchange-azure-blob>`, :ref:`Google Cloud Storage <fte-exchange-gcs>`, or a
```

Comment:
I would remove the mention of `fte-exchange-local-filesystem`. This implementation is built only for local testing and is not designed for production use cases.

Reply:
@sheajamba can you do that...

Reply:
or add something like "for testing"